scripts/ubuntu/hello-world-deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+export DEBIAN_FRONTEND=noninteractive
+
+apt update
+
+if ( which npm )
+then
+  echo "-> npm is installed"
+else
+  echo "-> Installing npm"
+  apt install npm -y
+fi
+
+if ( which node )
+then
+  echo "-> nodejs is installed"
+else
+  echo "-> Installing nodejs"
+  apt install nodejs -y
```

Comment:
If you check out the official install from the nodejs website there is a script you can call that allows you to install any version of nodejs you want and it includes npm.